packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -585,7 +599,8 @@ class Frame extends React.Component {
         ]);
 
         return (
-          <div data-terra-form-select-toggle className={toggleClasses} onMouseDown={this.handleToggleMouseDown}>
+          // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
Could you give some reasoning here for disabling this?

Reply:
The linter wants the div to have all of the key events (space/enter) that a button would have so it can be activated via the keyboard. This element cannot receive focus so the keyboard events wouldn't work either way. This was sneaking around the rule previously by using mouse down instead of click to avoid the rule, but the functionality/result is the same conceptually.

Reply:
What would you think about adding that comment to the file?

Reply:
Added some comments to the disables:

https://github.com/cerner/terra-core/pull/3066/commits/2c58713b37a0c2b29f07671ff04a1de239e6f471